src/main/java/com/bieganski/ox/Board.java

```diff
@@ -0,0 +1,49 @@
+package com.bieganski.ox;
+
+import java.util.TreeSet;
+
+class Board {
+    private int size;
+    private TreeSet<Field> fields;
+
+    Board(int size) {
+        fields = new TreeSet<>();
+        this.size = size;
+    }
+
+    void addFiled(Field field) throws FieldIsNotEmptyException {
```

Comment:
add field unless it's not empty... sounds strange. Please change appropriately. Naming is hard, but important after all.